chess-engine/Rules.cs

```diff
@@ -8,6 +8,75 @@ namespace chess_engine
+			new[]{new[]{28,35,42,49,56}, new[]{30,39}, new[]{14,7}, new[]{12,3} },	//21
+			new[]{new[]{29,36,43,50,57}, new[]{31}, new[]{15}, new[]{13,4} },	//22
+			new[]{new[]{30,37,44,51,58}, new[]{14,5} },	//23
+			new[]{new[]{33,42,51,60}, new[]{17,10,3}, },	//24
+			new[]{new[]{32}, new[]{34,43,52,61}, new[]{18,11,4}, new[]{16} },	//25
+			new[]{new[]{33,40}, new[]{35,44,53,62}, new[]{19,12,5}, new[]{17,8} },	//26
+			new[]{new[]{34,41,48}, new[]{36,45,54,63}, new[]{20,13,6}, new[]{18,9,0} },	//27
+			new[]{new[]{35,42,49,56}, new[]{37,46,55}, new[]{21,14,7}, new[]{19,10,1} },	//28
+			new[]{new[]{36,43,50,57}, new[]{38,47}, new[]{22,15}, new[]{20,11,2} },	//29
+			new[]{new[]{37,44,51,58}, new[]{39}, new[]{23}, new[]{21,12,3} },	//30
+			new[]{new[]{38,45,52,59}, new[]{22,13,4} },	//31
+			new[]{new[]{41,50,59}, new[]{25,18,11,4}, },	//32
+			new[]{new[]{40}, new[]{42,51,60}, new[]{26,19,12,5}, new[]{24} },	//33
+			new[]{new[]{41,48}, new[]{43,52,61}, new[]{27,20,13,6}, new[]{25,16} },	//34
+			new[]{new[]{42,49,56}, new[]{44,53,62}, new[]{28,21,14,7}, new[]{26,17,8} },	//35
+			new[]{new[]{43,50,57}, new[]{45,54,63}, new[]{29,22,15}, new[]{27,18,9,0} },	//36
+			new[]{new[]{44,51,58}, new[]{46,55}, new[]{30,23}, new[]{28,19,10,1} },	//37
+			new[]{new[]{45,52,59}, new[]{47}, new[]{31}, new[]{29,20,11,2} },	//38
+			new[]{new[]{46,53,60}, new[]{30,21,12,3} },	//39
+			new[]{new[]{49,58}, new[]{33,26,19,12,5}, },	//40
+			new[]{new[]{48}, new[]{50,59}, new[]{34,27,20,13,6}, new[]{32} },	//41
+			new[]{new[]{49,56}, new[]{51,60}, new[]{35,28,21,14,7}, new[]{33,24} },	//42
+			new[]{new[]{50,57}, new[]{52,61}, new[]{36,29,22,15}, new[]{34,25,16} },	//43
+			new[]{new[]{51,58}, new[]{53,62}, new[]{37,30,23}, new[]{35,26,17,8} },	//44
+			new[]{new[]{52,59}, new[]{54,63}, new[]{38,31}, new[]{36,27,18,9,0} },	//45
+			new[]{new[]{53,60}, new[]{55}, new[]{39}, new[]{37,28,19,10,1} },	//46
+			new[]{new[]{54,61}, new[]{38,29,20,11,2} },	//47
+			new[]{new[]{57}, new[]{41,34,27,20,13,6}, },	//48
+			new[]{new[]{56}, new[]{58}, new[]{42,35,28,21,14,7}, new[]{40} },	//49
+			new[]{new[]{57}, new[]{59}, new[]{43,36,29,22,15}, new[]{41,32} },	//50
+			new[]{new[]{58}, new[]{60}, new[]{44,37,30,23}, new[]{42,33,24} },	//51
+			new[]{new[]{59}, new[]{61}, new[]{45,38,31}, new[]{43,34,25,16} },	//52
+			new[]{new[]{60}, new[]{62}, new[]{46,39}, new[]{44,35,26,17,8} },	//53
+			new[]{new[]{61}, new[]{63}, new[]{47}, new[]{45,36,27,18,9,0} },	//54
+			new[]{new[]{62}, new[]{46,37,28,19,10,1} },	//55
+			new[]{new[]{49,42,35,28,21,14,7}, },	//56
+			new[]{new[]{50,43,36,29,22,15}, new[]{48} },	//57
+			new[]{new[]{51,44,37,30,23}, new[]{49,40} },	//58
+			new[]{new[]{52,45,38,31}, new[]{50,41,32} },	//59
+			new[]{new[]{53,46,39}, new[]{51,42,33,24} },	//60
+			new[]{new[]{54,47}, new[]{52,43,34,25,16} },	//61
+			new[]{new[]{55}, new[]{53,44,35,26,17,8} },	//62
+			new[]{ new[] { 54, 45, 36, 27, 18, 9 }, new[]{0}, },	//63
```

Comment:
Are you sure this is correct?
I would expect it to be:
new[]{ new[] { 54, 45, 36, 27, 18, 9, 0} },	//63
Why is it a separate array?